src/Generators/Statements/EventGenerator.php

```diff
@@ -36,7 +36,7 @@ public function output(Tree $tree): array
                     }
 
                     if (! $this->files->exists(dirname($path))) {
-                        $this->files->makeDirectory(dirname($path), 0755, true);
```

Comment:
Why are all these removed?

Reply:
makeDirectory has only 1 parameter if I'm not wrong?

Reply:
[You're wrong](https://www.php.net/manual/en/function.mkdir.php). I am surprised the tests passed for you when you removed it. Unless you removed it from both sides at the same time. 🙃